### ISSUE TITLE
MarkUs assignment upload

### ISSUE THREAD

**Issue by henrytwo** [karansher/computer-graphics-raster-images#11]
When will the MarkUs upload for the assignment be opened up?

**Comment by abhimadan** [comment]
We should have it up by the end of this week.

**Comment by ParthSindhu** [comment]
I think the Link would needs to to updated to https://markus.teach.cs.toronto.edu/2022-09 in the readme as well

**Comment by henrytwo** [comment]
@abhimadan @karansher 

There isn't an enforced list of required files to upload on MarkUs.
<img width="321" alt="image" src="https://user-images.githubusercontent.com/17518332/190682223-de30962b-62de-4bd9-b016-ccd7e25ba45d.png">

I assume we only need to upload the 12 `.cpp` files under `src` that we implemented, right?
<img width="343" alt="image" src="https://user-images.githubusercontent.com/17518332/190682158-57ae749a-4516-420b-adda-91bbdab59457.png">

**Comment by abhimadan** [comment]
Yes, just upload the cpp files in the src folder. We might update the submission setup so I'd recommend holding off on uploading your files for now.

**Comment by abhimadan** [comment]
Actually, disregard my earlier comment - feel free to submit now. We'll hold off on updating the submission.
